src/main/scala/cromwell/engine/io/gcs/GcsFileAttributes.scala

```diff
@@ -0,0 +1,25 @@
+package cromwell.engine.io.gcs
+
+import java.nio.file.Path
+import java.nio.file.attribute.{FileTime, BasicFileAttributes}
+
+class GcsFileAttributes(path: Path) extends BasicFileAttributes {
+
```

Comment:
I see your point, but I thought the distinction between "is just not implemented now ~ TODO" and "will not be supported" was made by `NotImplementedError` and `UnsupportedOperationException`.
I can add a short message in the exception though to say this is meant to be implemented when the time comes.